### ISSUE TITLE
Config "phpSniffer.snippetExcludeSniffs" not working

### ISSUE THREAD

**Issue by dimacros** [wongjn/vscode-php-sniffer#12]
**Config snippetExcludeSniffs Not Working**
My config:
```
{
    "phpSniffer.executablesFolder": "./vendor/bin/",
    "phpSniffer.standard": "PSR2",
    "phpSniffer.snippetExcludeSniffs": [
        "Squiz.Classes.ValidClassName.NotCamelCaps"
    ]
}
```
Everything works fine, except the exclude sniffs.

**Comment by wongjn** [comment]
The `snippetExcludeSniffs` option is for excluding sniffs when formatting a code snippet only — such as when formatting on paste or on the command `format on selection`. This is to avoid things like unnecessary new line insertions or anything else where full document context for the snippet is not available.

If you are trying to use `PSR2` but with `Squiz.Classes.ValidClassName.NotCamelCaps` excluded from the standard, you should [create a `phpcs.xml` file](/squizlabs/PHP_CodeSniffer/wiki/Annotated-Ruleset) and point `phpSniffer.standard` to it, something like the below:

phpcs.xml:

```xml
<?xml version="1.0"?>
<ruleset name="Custom Standard">
  <rule ref="PSR2">
    <exclude name="Squiz.Classes.ValidClassName.NotCamelCaps"/>
  </rule>
</ruleset>
```

Settings:
```json
{
    "phpSniffer.executablesFolder": "./vendor/bin/",
    "phpSniffer.standard": "./phpcs.xml",
}
```

**Comment by SharakPL** [comment]
So what exactly is `"phpSniffer.snippetExcludeSniffs"` for if I still have to create separate phpcs.xml file to exclude sniffs? How to ignore few rules eg. `PSR12.Files.FileHeader.SpacingAfterBlock` or `PSR1.Classes.ClassDeclaration.MissingNamespace`. I'm using PSR12 and only need to stop these rules from being reported:

![obraz](https://github.com/wongjn/vscode-php-sniffer/assets/5007456/9831f7ee-902c-4b0c-9b35-66ac06c18bc8)
